app/models/case/sar/offender.rb

```diff
@@ -239,6 +241,18 @@ def validate_third_party_address
     errors[:third_party_relationship].any?
   end
 
+  def validate_third_party_email
+    if (third_party || recipient == "third_party_recipient")
+      return if third_party_email.blank?
+
+      third_party_email.split =~ /\A([^@,]+)@([^@,]+)\z/ # regex disallows commas and additional @s
```

Comment:
I don't think you need to use split here.

Reply:
Removed split and refactored code